src/challenge.js

```diff
@@ -186,21 +195,19 @@ module.exports = (fcrToken, LMSR, web3, id, address, defaultOptions) => {
     if (!isStarted) {
       throw new Error('challenge has not been started')
     }
-
     const isFunded = await contract.methods.isFunded().call()
     if (!isFunded) {
       throw new Error('challenge markets have not been funded')
     }
 
     const categoricalEvent = await getCategoricalEvent()
-
+    const catAddr = await categoricalEvent.options.address;
```

Comment:
nitpicky, but should probably be more explicit here: `categoricalEventAddress`

Reply:
thanks!! was debugging! didn't mean to add that